packages/fxa-auth-server/lib/oauth/token.js

```diff
@@ -35,7 +36,23 @@ exports.getTokenId = async function getTokenId(accessToken) {
  * Verify an accessToken.
  */
 exports.verify = async function verify(accessToken) {
-  // JWT access tokens are still database backed, continue
+  // JWT tokens with lifespan < SHORT_ACCESS_TOKEN_TTL_IN_MS are not
+  // stored in the db
+  if (!validators.jwt.validate(accessToken).error) {
+    const t = await JWTAccessToken.verify(accessToken);
+    const lifespan = (t.exp - t.iat) * 1000;
```

Comment:
Oh, very nice use of `iat` to reconstruct the original rule, rather than just doing a "does this have `SHORT_ACCESS_TOKEN_TTL_IN_MS` left before it expires" 👍